core/slice.go

```diff
@@ -141,8 +141,18 @@ func (sl *Slice) Append(header *types.Header, domTerminus common.Hash, td *big.I
 		return sl.nilPendingHeader, nil, err
 	}
 
+	// If this was a coincident block, our dom will be passing us a set of newly confirmed ETXs
+	// If this is not a coincident block, we need to build up the list of confirmed ETXs using the subordinate manifest
+	if !isCoincident {
+		// TODO: need to collect these for all blocks since last coincident block with this location
```

Comment:
Is this TODO complete?

Reply:
oops, yes it is. Good catch. I'll remove it.

Reply:
Removed in force push